### DISCUSSION TITLE
Prefix with number +9

### DISCUSSION THREAD

**Discussion by DanaDovzh** [primefaces/community#1202] [PrimeNG] [answered]
This mask accepts the number 9 as formatting for entering numbers, but I need to have the number 9 prefixed to the phone number. However, when I enter 9, it simply reads it as part of the mask. How I can use this mask and have a code +998

```
 <p-inputMask 
  [unmask]="false"
  placeholder="+998 (__)___-____"
  mask="+998 (99)999-9999">
</p-inputMask>
```

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we'll address this issue in upcoming releases https://github.com/primefaces/primeng/issues/14833
